sensor_stack/seabotix/src/FuzzyControl/fuzzyParams.py

```diff
@@ -0,0 +1,75 @@
+f_ssets = [[ # error
```

Comment:
We can re-write this code later so as to use ROS' Parameter server to get these parameters.

The values are defined in a `yaml` file such as this one [config/dynamics_g500.yaml](https://github.com/uji-ros-pkg/underwater_simulation/blob/indigo-devel/underwater_vehicle_dynamics/config/dynamics_g500.yaml) and then accessed using a function in Python like [getParams](https://github.com/uji-ros-pkg/underwater_simulation/blob/indigo-devel/underwater_vehicle_dynamics/src/dynamics.py#L26)